examples/usecase-passport/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "keystone-app",
```

Comment:
this might need to be updated to "@keystone-6/example-custom-session-passportjs" - I also wonder if this is better placed in a `custom-seesion-passportjs` folder as it better fits with custom-session than it does with `usecase` - but @dcousens might have more thoughts on that.